docs/usage/self-hosted-configuration.md

```diff
@@ -128,7 +128,9 @@ e.g.
 
 ## composerIgnorePlatformReqs
 
-Set to `false` to prevent usage of `--ignore-platform-reqs` in the Composer package manager.s
+Set the platform requirements you want to exclude when using Composer. Each item will be added to the composer command with `--ignore-platform-req`. When set to an empty array, `--ignore-platform-reqs` will be added to the Composer command.
```

Comment:
```suggestion
Set the platform requirements you want to exclude when using Composer.
Each item will be added to the Composer command with `--ignore-platform-req`.
When set to an empty array, `--ignore-platform-reqs` will be added to the Composer command.
```

We use one sentence per line. 😉 

I'd like an explanation that covers _why_ you might want to ignore/force through some platform requirements when you use Composer in a certain way.

Reply:
Done!